lib/plugins/aws/package/compile/events/websockets/lib/routeResponses.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const _ = require('lodash');
+const BbPromise = require('bluebird');
+
+module.exports = {
+  compileRouteResponses() {
+    this.validated.events.forEach(event => {
+      if (!event.routeResponseSelectionExpression) {
+        return;
+      }
+
+      const websocketsRouteResponseLogicalId = this.provider.naming.getWebsocketsRouteResponseLogicalId(
+        event.route
+      );
+
+      const websocketsRouteLogicalId = this.provider.naming.getWebsocketsRouteLogicalId(
+        event.route
+      );
+
+      _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Resources, {
```

Comment:
Let's use `Object.assign` (which tend to resort to lodash, only when there's no convenient native counterpart, and here deep assign is not justified)